packages/api/test/e2e/api.calls.e2e.ts

```diff
@@ -101,14 +101,18 @@ describe('e2e api calls', () => {
 
   describe('Get session info', () => {
     it('Get correct session information (length, last length, era, current index, session per era', async () => {
-      const currentSession = await api.derive.session.info();
```

Comment:
api.derive.session.info() will fail because - it internally uses - api.query.staking.activeEra which does not exist on the chain..

Reply:
Fixed the above wth this https://github.com/cennznet/api.js/pull/264/commits/68f78e0c79879703190e8fce49b497d2749c6ab1

Reply:
Fixed this with https://github.com/cennznet/api.js/pull/264/commits/68f78e0c79879703190e8fce49b497d2749c6ab1